feature/metis/shared/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/shared/db/MetisDao.kt

```diff
@@ -140,7 +140,7 @@ interface MetisDao {
         postingType: BasePostingEntity.PostingType = BasePostingEntity.PostingType.STANDALONE
     )
 
-    @Query("delete from metis_post_context where client_post_id = :clientPostId")
+    @Query("delete from postings where id = :clientPostId")
```

Comment:
Why is the query different now?

Reply:
Good question, tbh I am not sure myself why this worked before. When using this method to delete answers it (understandably) caused the app to crash because of a foreign key error, as there now is a posting with the respective metis_post_context being deleted. I do not know why this worked previously for deleting standalone posts